sos/report/plugins/podman.py

```diff
@@ -44,7 +44,9 @@ def setup(self):
             'ps -a',
             'stats --no-stream --all',
             'version',
-            'volume ls'
+            'volume ls',
+            'unshare /proc/self/uid_map',
+            'unshare /proc/self/gid_map',
```

Comment:
These need to be run _by_ the rootless user to get the intended output:

```
[jake@terra sos]$ podman unshare cat /proc/self/uid_map
         0       1000          1
         1     100000      65536
[jake@terra sos]$ sudo podman unshare cat /proc/self/uid_map
Error: please use unshare with rootless

```

Reply:
Ah that was misunderstanding on my end. Wll fix along with other comments.